packages/cozy-sharing/src/components/ShareModal.jsx

```diff
@@ -30,64 +30,54 @@ export const ShareModal = ({
   sharingDesc,
   twoStepsConfirmationMethods
 }) => {
-  const showShareCozyToCozyForNotes =
-    flag('notes.sharing-cozy-to-cozy') !== null &&
-    flag('notes.sharing-cozy-to-cozy') &&
-    documentType === 'Notes'
+  const shareDialogOnlyByLink =
```

Comment:
I'm pretty sure I worked on this, so I'm not blaming anyone, but it's still too obscure to me.

Ideally, we should have some config object we act upon when we receive data.

Something like that for instance: 
```
const shareableOnlyByLink = ['notes', 'albums']

export const isShareableOnlyByLink = (type: DocumentType): boolean =>
  shareableOnlyByLink.includes(type)

isShareableOnlyByLink('notes') // true
```

Reply:
It would be more complicated to have a generic config object because the 'notes' app need a flag and the 'albums' app not. So I prefer to keep it like this. But it will definitively need some refactoring because there is some "onlyByLink" content in "cozyToCozy" part.